doc/man5/x509v3_config.pod

```diff
@@ -22,6 +23,9 @@ Each entry in the extension section takes the form:
 
 If B<critical> is present then the extension will be marked as critical.
 
+If multiple entries are processed for the same extension name,
+later entries override earlier ones with the same name.
```

Comment:
Ah, now I get it. However this seems to me as substantial change in the processing. Perhaps at least CHANGES.md entry should be added?

Reply:
This is basically just a clarification in the code (at points like the one referred to above)
and rather prominently at this point in the documentation.
Further down (where users hardly will read it) it already said before:
```
OpenSSL does not support multiple occurrences of the same field within a
section. In this example:

 [extensions]
 subjectAltName = @alt_section

 [alt_section]
 email = steve@here
 email = steve@there

will only recognize the last value.
```
Note that this is not 'just' an OpenSSL restriction but also due to an RFC 5280 requirement.
The intro of section 4.2 says:
> A certificate MUST NOT include more than one instance of a particular extension.

So I don't really see the need for an addition to CHANGES.md on that, 
but I can do if you prefer to have one - if so, how do you suggest to phrase it?

Reply:
OK, then no CHANGES.md are needed, but could you please amend the commit message to explain the change more?

Reply:
Good point - done.